js/app.jsx

```diff
@@ -486,6 +486,26 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      var lines = [];
+      if (str) {
+        lines = str.split(/,|ÿ/)
+                   .map(val => { return val.trim(); })
```

Comment:
You can say `val => val.trim()` but really I'd just do it in the next block and only have one `map`.